app/frontend/css/application.scss

```diff
@@ -88,14 +88,25 @@ $govuk-global-styles: true; // https://frontend.design-system.service.gov.uk/sas
     }
   }
 
-//defra header
 .defra {
 
+  &-responsive {
```

Comment:
I don;t think this is working exactly how Nick wants it but we can iterate in a later PR.